data/vaccineData/2021-10-30.json

```diff
@@ -1,221 +1,221 @@
 {
   "metadata": {
-    "createdAtUtcTimeIso": "2021-11-07T05:46:24.787Z",
-    "updatedAtUtcTimeIso": "2021-11-07T05:46:24.787Z"
+    "createdAtUtcTimeIso": "2021-11-13T08:14:28.980Z",
+    "updatedAtUtcTimeIso": "2021-11-13T08:14:28.980Z"
```

Comment:
:( Should these have changed?

Reply:
yea the files get completely regenerated. It deletes all the stored files and then recreates them. 

I could make the script only update the data if its different if ya prefer?

We didn't really use the created at and updated at date for much other than the last updated date on the site soo I didn't right a great regen script. Can do though if we want.

Reply:
Yeh what do you think? I mean technically this is a re-intepretation of the same old data. Maybe the updated / created time should live with the raw html?

It may be an issue if we try and create pages for previous dates 🤔

Reply:
Yea I agree it should live with the html. Will have a look moving the date there somehow. 

Although I doubt we'd ever use the date for previous day's. Unless we wanna tell the user what time we fetched the data on. Which seems weird.